src/ArduinoLogger.h

```diff
@@ -464,6 +464,21 @@ class LoggerBase
 		}
 	}
 
+	// Returns the amount of data being held by the ready buffer
+	// Must implement in the derived class
+	virtual size_t ready_buffer_internal_size() const noexcept
+	{
+		return size();
```

Comment:
Shouldn't this return 0 seeing as ready_buffer_exists() returns false?

Reply:
Changed to return 0